frontends/mit-learn/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -21,6 +22,10 @@ type SearchChip = {
   icon?: React.ReactElement
 }
 
+const HERO_IMAGE_IDX = _.shuffle([1, 2, 3, 4, 5])
+
+const HERO_IMAGE_URL = `/static/images/hero/hero-${HERO_IMAGE_IDX[0]}.png`
```

Comment:
I'm ok skipping a test for the randomization, but some comments about testing this.

1. A global declaration + randomization may have some downsides:
    - we can't easily test this in Jest—even if we thought it was worth it—because you'd have to re-import the module to re-randomize, and module imports are generally cached. 
    - ...which leads me to: We've been discussing NextJS and server-side rendering lately. I'm not 100% sure, but I expect that this also would not randomized if served by NextJS, because the randomization is done globally on import. So you'd probably have to restart the server to randomize.
    - Both of the above could be solved by doing the randomization like `const getRandomHeroImage = () => { ... }` and throwing it in a `useEffect(getRandomHeroImage, [])`—that would run once when the component is mounted.
        - but even if you did that, maybe we would cache server-generated HTML, so....
2. Even if you made the change above, any test in Jest would feel pretty artificial to me. I'd much rather have a browser-based test. (E.g., hit "refresh" with playwright 5 times and check that the images aren't all the same. It'll fail some fraction—1/625th— of the time like Carey mentions because of repeats, but whatever.)

Which is all a longwinded way of saying... I don't think it's worth adding a test right now.